### PR TITLE
Support llm-awq backend

### PR DESCRIPTION
**Pull request by Romanticoseu**
## Description

Support llm-awq backend. Based on #9849 

### 3. Summary of the change 
Support llm-awq backend-->

### 4. How to test?
- [x] Application test

### 5. New dependencies
None

### CODE REVIEW

**Comment by Romanticoseu** [comment]
## Application test:
Sample output of https://huggingface.co/ybelkada/vicuna-7b-1.5-awq#9856 
``` bash
-------------------- Prompt --------------------
### HUMAN:
Write a python code to find Fibonacci

### RESPONSE:

-------------------- Output --------------------
### HUMAN:
Write a python code to find Fibonacci

### RESPONSE:

Here's a Python code to find the Fibonacci sequence:
```python
def fibonacci(n):
    if n <= 1:
        return n
    else:
        return fibonacci(n-1) + fibonacci(n-2)

for i in range(10):
    print(fibonacci(i))
```

File: python/llm/example/CPU/HF-Transformers-AutoModels/Advanced-Quantizations/AWQ/README.md [hunk 1]
```diff
@@ -37,6 +41,17 @@ pip install accelerate==0.24.1
 pip install einops
 ```
 
+**(Optional) Support for LLM-AWQ Backend**
+
+BigDL-LLM uses `autoawq` as its default awq backend, and also automatically supports `llmawq` if your model is orginally quantized by llmawq. You can directly input an llmawq model in the same way as autoawq, after installing the following dependencies:
+
+```bash
+git clone https://github.com/mit-han-lab/llm-awq
```
Comment (Uxito-Ada) [inline python/llm/example/CPU/HF-Transformers-AutoModels/Advanced-Quantizations/AWQ/README.md:49]: Do we really require users to install llm-awq? It seems nothing is imported from llm-awq in codes.

Comment (Romanticoseu) [reply python/llm/example/CPU/HF-Transformers-AutoModels/Advanced-Quantizations/AWQ/README.md:49]: You are correct, we have integrated the `llm-awq` code.

**Review by cyita — APPROVED** [review]
LGTM

**Pull request merged**